workflow-ui/core-android/src/main/java/com/squareup/workflow/ui/BuilderBinding.kt

```diff
@@ -36,8 +36,8 @@ import kotlin.reflect.KClass
  *      companion object : ViewBuilder<MyScreen>
  *      by BuilderBinding(
  *          type = MyScreen::class,
- *          builder = { _, initialRendering, context, _ ->
- *            MyView(context).apply {
+ *          viewConstructor = { initialRendering, _, context, _ ->
+ *            MyView(context, null).apply {
```

Comment:
Why the explicit `null`?

Reply:
The `MyView` class has an `AttributeSet?` and I wasn't sure where to get a non-null from.

```
class MyView(
  context: Context,
  attributeSet: AttributeSet?
) 
```

Reply:
Oh, good catch. How about giving `attributeSet` a default value, so:

```kotlin
 *    class MyView(
 *      context: Context,
 *      attributeSet: AttributeSet? = null
 *    ) : FrameLayout(context, attributeSet) {
```

Reply:
I like @SalvatoreT's suggestion of removing it entirely – if this view is only used by this binding, it never uses it, right? It would keep the sample simpler too.